### DISCUSSION TITLE
蘋果M3處理器

### DISCUSSION THREAD

**Discussion by uwje** [1101-GS4719-C/Python-Community-GS4719-C#1683] [General]
在前天，平鍋發布了基於arm架構的m3晶片的電腦，因為在天前高通也發布了一樣基於arm架構的晶片，所以難免被做比較。
我認為兩個個有優缺，蘋果的矗立器強大在低功耗下也能有強大的效能，還優基於apple生態系統的配合，也是apple的強項，而高通的則在於它搭載windows系統，所以可以獲得更多大廠的資源。我個人也認為知單看跑分的確差異不大，但如何完整的江跑分轉化為實際使用體驗也是很重要的。
參考資料:https://www.cool3c.com/article/202491

**Comment by Terryyyyyyyyyyyy** [comment]
我認為也需要關注記憶體的讀取速度是否跟m2系列一樣被降速，但我認為以影音剪輯來說，他依然會是不錯用的電腦，雖然價格有點偏高，但光可以不插電下仍可全速運轉這點就不知道打趴多少人，究其原因便是其優秀的能耗比。

**Comment by mrweixd** [comment]
頻果最大的缺點在於它的價錢⋯⋯但是續航力是真的強。

**Comment by Ryanlolry** [comment]
蘋果的續航是因為使用arm架構處理器，使其的能耗比大量領先使用x86架構的處理器，而且再加上他不用支援多家廠商的軟體適配，他只需保留最近期的更新就好，能有效減少軟體在運作時的速度。
多說一句：價錢較高不是蘋果的缺點，而是你的缺點

**Comment by davidteng00000** [comment]
蘋果M3與高通ARM晶片各有優勢。蘋果M3以低功耗提供強大效能，並融入蘋果生態系統；高通ARM搭載Windows系統，可獲得更多資源。除跑分外，實際使用體驗也是至關重要，不能只看跑分。
